pyproject.toml

```diff
@@ -29,7 +29,9 @@ dependencies = [
     "click>=7.0,<9.0.0",
     "importlib-metadata>=4.0",
     "pygithub>=1.59.1",
+    "inflect>=7.0.0",
     "lxml>=4.9.3",
+    "num2words>=0.5.0",
```

Comment:
Hmm I htought we were not using it

Reply:
Good catch, I forgot to remove this one
Thanks!